tests/test_schema.py

```diff
@@ -778,6 +854,143 @@ def test_authentication(self):
+
+        self.user = get_user_model().objects.create(username='test')
+        self.context_value = RequestFactory().get(GRAPHQL_ENDPOINT)
+        self.context_value.user = self.user
+
+        self.proj = Project(name='Example',
+                            title='Example title')
+        self.proj.save()
+
+    def test_project(self):
+        """Check if it returns the list of projects"""
+
+        client = graphene.test.Client(schema)
+        executed = client.execute(BT_PROJECTS_QUERY,
+                                  context_value=self.context_value)
+
+        projects = executed['data']['projects']['entities']
+        self.assertEqual(len(projects), 1)
+
+        proj = projects[0]
+        self.assertEqual(proj['id'], str(self.proj.id))
+        self.assertEqual(proj['name'], 'Example')
+        self.assertEqual(proj['title'], 'Example title')
+        self.assertEqual(proj['parentProject'], None)
+        self.assertEqual(proj['ecosystem'], None)
+
+    def test_filter_registry(self):
+        """Check whether it returns the project searched when using filters"""
+
+        client = graphene.test.Client(schema)
+        test_query = BT_PROJECTS_QUERY_FILTER % (self.proj.id, 'Example')
+        executed = client.execute(test_query,
+                                  context_value=self.context_value)
+
+        projects = executed['data']['projects']['entities']
+        self.assertEqual(len(projects), 1)
+
+        proj = projects[0]
+        self.assertEqual(proj['id'], str(self.proj.id))
+        self.assertEqual(proj['name'], 'Example')
+        self.assertEqual(proj['title'], 'Example title')
+        self.assertEqual(proj['parentProject'], None)
+        self.assertEqual(proj['ecosystem'], None)
+
+    def test_filter_non_existing_registry(self):
+        """Check whether it returns an empty list when searched with a non existing project"""
+
+        client = graphene.test.Client(schema)
+        test_query = BT_PROJECTS_QUERY_FILTER % (11111111, 'Ghost-Project')
+        executed = client.execute(test_query,
+                                  context_value=self.context_value)
+
+        projects = executed['data']['projects']['entities']
+        self.assertListEqual(projects, [])
+
+    def test_pagination(self):
+        """Check whether it returns the projects searched when using pagination"""
+
+        # Creating additional projects
+        proj1 = Project(name='Example-1',
+                        title='Example title 1')
+        proj1.save()
+
+        proj2 = Project(name='Example-2',
+                        title='Example title 2')
+        proj2.save()
+
+        client = graphene.test.Client(schema)
+        test_query = BT_PROJECTS_QUERY_PAGINATION % (1, 2)
+        executed = client.execute(test_query,
+                                  context_value=self.context_value)
+
+        projects = executed['data']['projects']['entities']
+        self.assertEqual(len(projects), 2)
```

Comment:
You should try here if the second page returns different projects. If not, pagination is not working.